src/services/gapi.service.js

```diff
@@ -0,0 +1,22 @@
+import config from 'config';
+import axios from 'axios';
+
+const { apiVersion } = config;
+const gapiBaseUrl = 'https://classroom.googleapis.com/v1';
+
+const getStudentCourses = (token) => {
+  return axios({
```

Comment:
can we use services/http.service to handle all Axios request?

e.g
const create = (projectId, playlist) => httpService
  .post(`/${apiVersion}/projects/${projectId}/playlists`, playlist)
  .then(({ data }) => data)
  .catch((err) => Promise.reject(err.response.data));

Reply:
The http.service is tailored to requests directed at our own API. This service calls on the google classroom API so I made it standalone.